src/include/cvc5_private.h

```diff
@@ -17,7 +17,8 @@
 #ifndef CVC5_PRIVATE_H
 #define CVC5_PRIVATE_H
 
-#if !(defined(__BUILDING_CVC5LIB) || defined(__BUILDING_CVC5LIB_UNIT_TEST))
+#if !(defined(__BUILDING_CVC5LIB) || defined(__BUILDING_CVC5PARSERLIB) \
```

Comment:
What was the issue here?

Reply:
We need to be able to include internal stuff when compiling the parser library, since it uses internal utilities (mostly for printing commands).